packages/vuetify/types/services/icons.d.ts

```diff
@@ -13,7 +13,7 @@ export interface VuetifyIconOptions {
    * fa: <a href="https://fontawesome.com/get-started/web-fonts-with-css">FontAwesome 5</a>
    * fa4: <a href="">FontAwesome 4</a> TODO: link
    */
-  iconfont: 'md' | 'mdi' | 'fa' | 'fa4' // TODO: camelCase
+  iconfont: 'mdi' | 'mdi-svg' | 'md' | 'fa' | 'fa4' // TODO: camelCase
```

Comment:
There is no relation between that and mdi. It works with any svg path. Imho it should be `svg`.